test/specialize.lean

```diff
@@ -0,0 +1,10 @@
+import tactic.specialize
+
+example (h : ∀ x > 0, true) : true :=
+begin
+  specialize' h 1 _,
```

Comment:
Can you add a test with multiple underscores?

Reply:
Yup, and I'll add one with multiple goals.